src/components/PageNotFound/PageNotFound.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+// Styles
+import './PageNotFound.scss';
+
+export default () => {
+  return (
+    <div className="pagenotfound-container">
+      <h1 className="pagenotfound-container__head">Page Not Found</h1>
+      <h4 className="pagenotfound-container__text">
+        Oops! The page you are looking for has been removed or relocated.
+      </h4>
+      <div className="pagenotfound-container__button">
+        <Link to="./">Go Back</Link>
```

Comment:
I think this needs logic, if there is history entry, make a back button, otherwise send to homepage.

Reply:
That somewhat makes sense. But I think checking the history is not the right thing here. Consider the case where the history entry is itself not a valid path in our router. Example -

```
Path - webpack.js.org/abc/xyz 

History - ['abc', 'xyz']
```

Both the history entries are not valid paths as per our router. It would be difficult to maintain a set of valid paths and to check them for this feature. Also looking at how this is implemented in other products, this seems to be in the right direction.